stream-chat-android/src/main/java/com/getstream/sdk/chat/view/dialog/MessageMoreActionDialog.kt

```diff
@@ -73,6 +73,7 @@ internal class MessageMoreActionDialog(
                 dismiss()
             }
         }
+        binding.flagMessageButton.isVisible = style.flagMessageActionEnabled
```

Comment:
This change now shows the Flag action on our own messages as well (if the action is enabled), overriding the `visibility = GONE` call a couple lines above